docs/examples/distributed/003_multi_node/job.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#SBATCH --gpus-per-task=rtx8000:1
+#SBATCH --cpus-per-task=1
```

Comment:
Shouldn't this this be like in 002?
```suggestion
#SBATCH --cpus-per-task=4
```